pages/api/patients/update.ts

```diff
@@ -0,0 +1,45 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      let { user_id, pickup_enabled, dob, prescriptions } = req.body.data
+
+      var patient = null
+      if (!prescriptions) {
+        patient = await prisma.patient.update({
+          where: {
+            userId: user_id
+          },
+          data: {
+            pickupEnabled: pickup_enabled,
+            dob: dob
+          }
+        })
+      } else {
+        patient = await prisma.patient.update({
+          where: {
+            userId: user_id
+          },
+          data: {
+            pickupEnabled: pickup_enabled,
+            dob: dob,
+
+            prescriptions: prescriptions
+          }
+        })
+      }
```

Comment:
same comment as before regarding consolidating this conditional statement, lmk if I am missing something

Reply:
see comment above

Reply:
ah i missed the catch from above thanks!